conanfile.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
```

Comment:
We should add a comment about this being supported on a best-effort basis. We're okay with adding this to our repository, but since we don't use Conan, we won't know if it stops working, and we may need community members to help us debug problems that arise.

Reply:
No problem, you could invoke any Conan member to help you, including me.